patch_files/4_6_general_refactors_to_namespacing_process.json

```diff
@@ -0,0 +1,159 @@
+{
+    "affected_services": [
+        "cloudify-mgmtworker", 
+        "cloudify-restservice"
+    ], 
+    "community": false, 
+    "description": "Node templates relationships overloading support.", 
+    "manager_versions": [
+        "4.6"
+    ], 
+    "patch_version": "2.0.0", 
+    "patches": [
+        {
+            "destinations": [
+                "/opt/manager/env/lib/python2.7/site-packages/dsl_parser/utils.py", 
+                "/opt/mgmtworker/env/lib/python2.7/site-packages/dsl_parser/utils.py", 
+                "/opt/cfy/embedded/lib/python2.7/site-packages/dsl_parser/utils.py"
+            ], 
+            "patch_file": "4_6_general_refactors_to_namespacing_process_dsl_parser_utils.py", 
+            "sha256sum": "ac815577ce1f47f18cbd060fc77aefbe45a7cb282cdfda9e703486b297cb7a21"
+        }, 
+        {
+            "destinations": [], 
```

Comment:
entries with an empty `destinations` can well be removed, and the patch files dropped

Reply:
Done